decentraland/unity-renderer
# refactor: (sdk6) remove json for component data and add protocol-buffer

This branch can be previewed at:

https://play.decentraland.org/?explorer-branch=refactor/sdk-component-protobuf-enqueue
https://play.decentraland.zone/?explorer-branch=refactor/sdk-component-protobuf-enqueue

# How to test

Do full regression for SDK6 related mechanics. 
1. Check Genesis Plaza and its surroundings, all its `secondary games
2. Check Scenes related content (user-generated) and interactions with it: animations, UI's, materials, Raycasts, sounds
3. Check some other games (Poker maybe)
